docs/components/modeler/web-modeler/play-your-process.md

```diff
@@ -92,6 +92,38 @@ Play's rewind operation currently does not support the following elements:
 - Play rewinds processes by initiating a new instance and executing each element. However, if any element behaves differently from the previous execution, such as a Connector returning a different result, the rewind may fail.
 
+## Scenarios
+
+You can validate your process by creating scenarios for different paths, ensuring that it continues to function as expected after any changes to your diagram. Scenarios let you replay and confirm that the process completes correctly with the pre-defined actions and variables.
+
+### Saving a scenario
+
+After executing a path in your process, you can save it as a scenario by clicking the **Save scenario** button in the process instance header. To view your saved scenarios, navigate to the process definition page by clicking the **View all** button under the Scenarios column in the process instance header.
+
+![save scenario](img/play-save-scenario.png)
+
+### Scenario coverage
+
+Scenario coverage is calculated as the percentage of flow nodes in your process that are covered. This includes all elements, events, and gateways. For example, if 8 out of 10 flow nodes are covered, the coverage will be 80%.
+On the process definition page, covered paths are highlighted in blue. You can also click on individual scenarios to view their specific coverage.
+
+![save scenario](img/play-coverage.png)
+
+Once a process instance is completed, the process instance header will display how much your process scenario coverage would increase if this path were saved as a scenario.
+
+### Running a scenario
+
+Scenarios can be run from the process definition page using the **Run all scenarios** button, or by clicking the **Run scenario** button with the play icon for each individual scenario. The result of a scenario execution will be marked as **Completed** or **Failed.** Failed scenarios must be manually updated by clicking the **manually complete and update the scenario** button, especially when changes are made to your diagram that require further user input—such as when a new flow node is added to a previously saved scenario path.
+
+![save scenario](img/play-scenario-runs.png)
+
+### Limitations
+
+- Scenarios are stored in the browser's local storage, making them accessible only in the current browser and not usable outside of Play or in a different browser.
+- Call activities are not supported; therefore, scenarios containing them cannot be executed successfully.
+- Scenario paths that include process modifications are not supported.
+- Similar to process instances, scenarios do not run in isolation. For instance, if two scenario paths are defined for a process and both contain the same message event or signal event, running these scenarios simultaneously may lead to unintended consequences. Publishing a scenario or broadcasting a signal could inadvertently impact the other scenario, resulting in the failure of both.
```

Comment:
💭 It could be nice to include a link to the documentation for the nature of message events and signal events, where the issue with global publish/broadcast is mentioned. However, I am not aware of such a link at this moment.